packages/node/src/jsonLayout.ts

```diff
@@ -1,20 +1,21 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 
-import correlationId from "correlation-id";
+import correlationIdLib from "correlation-id";
 import stringify from "json-stringify-safe";
 import log4js, { LoggingEvent } from "log4js";
 import util from "util";
 
 function jsonFormat(logEvent: LoggingEvent) {
-    // The correlation id is undefined when not set in the current call stack
-    const correlation = correlationId.getId();
-    const json = {
+    const json: any = {
         "@t": logEvent.startTime,
         "@mt": util.format(...logEvent.data),
-        "@l": logEvent.level.levelStr,
-        correlationId: correlation
+        "@l": logEvent.level.levelStr
     };
 
+    // The correlation id is undefined when not set in the current call stack
```

Comment:
I removed this comment on purpose, I don't think it is important for the reader to know why it is undefined.